frontend/simple/views/SignUp.vue

```diff
@@ -17,17 +17,18 @@
             <h2 class="subtitle">Sign Up</h2>
 
             <p class="control has-icon">
-              <input v-form-ctrl class="input" name="name" pattern="\S*" placeholder="username" required>
+              <input class="input" name="name" v-validate data-rules="required|regex:^\s*\S+\s*$" placeholder="username" required>
```

Comment:
Oh, I see, you're right, I forgot to include the start/end anchors. Then the simplified regex should be `^\S+$`. Pretttty sure on that, but double check just in case.